apps/src/blockly/themes/cdoAccessible.js

```diff
@@ -0,0 +1,103 @@
+import GoogleBlockly from 'blockly/core';
+
+// The colors here come Martin Krzywinski's 24-color palette for colorblindness:
+// http://mkweb.bcgsc.ca/colorblind/palettes.mhtml#15-color-palette-for-colorbliness
+// http://mkweb.bcgsc.ca/colorblind/palettes/24.color.blindness.palette.txt
+// Each color in our standard palette was mapped to a new "nearest" color from the safe palette.
+// For now, this is a manual process but we could also automate this using:
+// https://github.com/dtao/nearest-color
+
+// Intentionally overriden styles from Google Blockly.
+const coreBlocklyOverrides = {
+  colour_blocks: {
+    colourPrimary: '#009FFA'
+  },
+  list_blocks: {
+    colourPrimary: '#6B069F'
+  },
+  logic_blocks: {
+    colourPrimary: '#005FCC'
+  },
+  loop_blocks: {
+    colourPrimary: '#FF2E95'
+  },
+  math_blocks: {
+    colourPrimary: '#00489E'
+  },
+  procedure_blocks: {
+    colourPrimary: '#009503'
+  },
+  text_blocks: {
+    colourPrimary: '#009175'
+  },
+  variable_blocks: {
+    colourPrimary: '#AB0D61'
+  }
+};
+
+// Used for Sprite Lab (and all related level subtypes), and Dance.
+const spriteLabAccessibleStyles = {
+  behavior_blocks: {
+    colourPrimary: '#00AF8E'
```

Comment:
Some duplicates were allowed such as colors for `behavior_blocks` and `flow_blocks`. These styles are used in lab-specific blocks and are mutually exclusive.

Reply:
Thanks for explaining this! It's apparent how much time and care you put into selecting these colors.